object/merge.test.ts

```diff
@@ -0,0 +1,121 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import merge from "./merge.ts";
+
+describe("merge", () => {
+  it("should merge with empty object", () => {
+    const a: object = {};
+    const b: object = { a: 1 };
+
+    const received = merge(a, b);
+    const expected: object = { a: 1 };
+
+    expect(received).toEqual(expected);
+  });
+
+  it("should merge and override the same property", () => {
+    const a: object = { a: 1, b: 3 };
```

Comment:
Please remove explicit typings when not needed (in this case it is clearly visible these are objects).